### PR TITLE
Adds better information for using kubectl inside minikube

### PR DESCRIPTION
**Pull request by Harkishen-Singh**
Signed-off-by: Harkishen-Singh <harkishensingh@hotmail.com>

Fixes: #7642

### CODE REVIEW

**Comment by afbjorklund** [comment]
Not sure I agree with "workaround", but I guess it is the best we can do here since we don't agree on the recommendation.

File: site/content/en/docs/handbook/kubectl.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+weight: 2
+description: >
+  Use kubectl inside minikube
+aliases:
+  - /docs/kubectl/
+---
+
+By default, kubectl gets configured to access the kubernetes cluster control plane
+inside minikube when the `minikube start` command is executed. 
+
```
Comment (afbjorklund) [inline site/content/en/docs/handbook/kubectl.md:12]: Maybe add a link to the kubectl documentation here as well ?

https://kubernetes.io/docs/tasks/tools/install-kubectl/

Comment (Harkishen-Singh) [reply site/content/en/docs/handbook/kubectl.md:12]: I don't think that might be the best place for the link. Would be better to have that in the last of the document. Is that agreeable?

Comment (Harkishen-Singh) [reply site/content/en/docs/handbook/kubectl.md:12]: Like, `For more help related to kubectl, visit the [docs](https://kubernetes.io/docs/tasks/tools/install-kubectl/).`

Comment (Harkishen-Singh) [reply site/content/en/docs/handbook/kubectl.md:12]: Got it. Done!

**Comment by afbjorklund** [comment]
If you want to add additional tips to the documentation, you could add the "alias" as in #7289

It would need the regular warnings and caveats, for people using inferior shells and such...

But avoids having to type out the full `minikube kubectl --` every time:

```bash
alias kubectl='minikube kubectl --'
```

**Comment by Harkishen-Singh** [comment]
> If you want to add additional tips to the documentation, you could add the "alias" as in #7289

That would be great!

**Comment by Harkishen-Singh** [comment]
> If you want to add additional tips to the documentation, you could add the "alias" as in #7289

That would be great! However, according to that issue, do we aim to remove the `--` support? If yes, then at that point, we need to change this doc as well.

**Comment by afbjorklund** [comment]
I'm not sure it is possible to change the flag parsing of subcommands, without changing a lot of minikube internals (cobra/viper)

However, if the user don't have to type it anyway - does it really matter ? We _could_ have a command to print out the path as well.
If you prefer running `~/.minikube/cache/linux/v1.18.3/kubectl` over the wrapper (which does handle os and version for you)
Adding that directory to the PATH would be pretty much the same as "installing" it. Well, without the `sudo` requirement that is.

**Comment by afbjorklund** [comment]
Typically, the logging of the path to the `kubectl` binary was broken: #8521

**Comment by Harkishen-Singh** [comment]
@afbjorklund for now, I have added the `alias` as a tip for easier usage.

**Pull request merged**